pod/perlfunc.pod

```diff
@@ -1345,9 +1345,9 @@ Portability issues: L<perlport/chroot>.
 =for Pod::Functions declare a separate global namespace that is an object class
 
 Declares the BLOCK or the rest of the compilation unit as being in the given
-namespace, which implements an object class.  This behaves similarly to
-L<C<package>|/package NAMESPACE>, except that the newly-created package behaves
-as a class.
+namespace, which implements an object C<class> (see L<perlclass>).  This
```

Comment:
I don't like the `C< >` around "class". This turns it from a description in English ("the `class` keyword ... implements an object class") to a self-referential definition ("the `class` keyword ... implements the `class` keyword").

Reply:
I'm not wedded to it.  I added the `C<>` out of parallelism with `package` in the same paragraph.  But let's hear what other people think.

Reply:
"package" is used twice here:

> This behaves similarly to `package` *[i.e. the "package" keyword]*, except that the newly-created package *[= English, not a keyword]* behaves as a class.

Adding code markup to the second occurrence of "package" (i.e. "... the newly-created `package`") would be misleading, too, because it implies the keyword itself is newly created IMHO.

Reply:
Can you suggest another way to address the concern raised in GH #22350?  (The first concern raised there; I'm not addressing the second.)

Reply:
From what I can see the "class" being `C<>` here is the IT concept (or perl concept) of a class, not the keyword.  I don't think it should not have the `C<>` around it.

The `L<perlclass>` itself is the link requested by #22350, and is fine.

Reply:
> From what I can see the "class" being `C<>` here is the IT concept (or perl concept) of a class, not the keyword. I don't think it should not have the `C<>` around it.
> [snip]

You have a double negative in that last sentence: *don't ... should not* ....  Can you clarify?  Thanks.

Reply:
@khwilliamson, ^^

Reply:
I think the C<> around class should be removed